quickcheck-contractmodel/quickcheck-contractmodel.cabal

```diff
@@ -89,6 +89,6 @@ library
       quickcheck-threatmodel
     -- Plutus and Cardano dependencies
     build-depends:
-      cardano-api                                     >= 8.2.0 && < 8.3,
-      cardano-ledger-core                             >= 1.2.0 && < 1.3,
-      cardano-ledger-shelley                          >= 1.2.0 && < 1.3,
+      cardano-api                                     >= 8.8.0 && < 8.9,
```

Comment:
Why can't we keep the lower bound at `8.2.0`?

Reply:
I.e. why can't we just bump all the upper bounds and keep the lower bounds low here?

Reply:
Very good point! I didn't think about that at all, and just let cabal generate the bounds. I will fix that.